### PR TITLE
Refactor for code style and readability

### PR DESCRIPTION
**Pull request by Dalamar42**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java [hunk 1]
```diff
@@ -74,40 +84,49 @@ public int process(PaChannelData channelData, Set<String> currentlyProcessing) {
                                 hierarchy.getItem().getCanonicalUri()
                         );
                     }
+
                     processed++;
                 } catch (Exception e) {
-                    log.error(String.format("Error processing channel %s, prog id %s",
-                            channel.getKey(), programme.getProgId()));
+                    log.error(format("Error processing channel %s, prog id %s",
```
Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: Err. Just use slf4j's formatting. This is stupid.

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java [hunk 2]
```diff
@@ -74,40 +84,49 @@ public int process(PaChannelData channelData, Set<String> currentlyProcessing) {
...
+                            channel.getKey(), programme.getProgId()
+                    ));
                 } finally {
                     unlock(currentlyProcessing, programmeLock);
                 }
             }
         } catch (Exception e) {
-            //TODO: should we just throw e?
-            log.error(String.format("Error processing channel %s", channel.getKey()), e);
+            log.error(format("Error processing channel %s", channel.getKey()), e);
```
Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: Same.

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java [hunk 3]
```diff
@@ -74,40 +84,49 @@ public int process(PaChannelData channelData, Set<String> currentlyProcessing) {
...
                     channel,
-                    channelData.scheduleDay());
+                    channelData.scheduleDay()
+            );
 
             scheduleVersionStore.store(channel, channelData.scheduleDay(), channelData.version());
         } catch (Exception e) {
-            log.error(String.format("Error trimming and writing schedule for channel %s", channel.getKey()), e);
+            log.error(
+                    format(
```
Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: Ditto here.

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java [hunk 4]
```diff
@@ -123,18 +142,19 @@ private void writeContent(Set<ContentHierarchyAndSummaries> hierarchies, Channel
                 }
             }
         } catch (Exception e) {
-            log.error(String.format("Error writing content for channel %s", channel.getKey()), e);
+            log.error(format("Error writing content for channel %s", channel.getKey()), e);
```
Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: And here.

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java [hunk 5]
```diff
@@ -144,6 +164,101 @@ private void lock(Set<String> currentlyProcessing, String programmeLock) throws
     }
 
     private String lockIdentifier(ProgData programme) {
-        return Strings.isNullOrEmpty(programme.getSeriesId()) ? programme.getProgId() : programme.getSeriesId();
+        return Strings.isNullOrEmpty(programme.getSeriesId())
+               ? programme.getProgId()
+               : programme.getSeriesId();
+    }
+
+    public interface ProcessorStep {
```
Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: oh you

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java (hunk 1 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: Gagh, didn't see, fixing

File: src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java (hunk 5 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/pa/PaChannelProcessor.java]: :P

**Pull request merged**